modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/types.ts

```diff
@@ -159,6 +168,10 @@ export type TestrayFactorOption = {
 	r_factorCategoryToOptions_c_factorCategory: TestrayFactorCategory;
 };
 
+export type TestrayIssue = {
+	name: string;
+};
```

Comment:
Need to bring more information
```suggestion
export type TestrayIssue = {
	name: string;
};
```